autoload/ruby_hl_lvar.vim

```diff
@@ -66,7 +66,7 @@ function! s:redraw_window()
 		endif
 	else
 		if get(b:, 'ruby_hl_lvar_match_pattern', '') != ''
-			let w:ruby_hl_lvar_match_id = matchadd(g:ruby_hl_lvar_hl_group, b:ruby_hl_lvar_match_pattern, 0)
+			let w:ruby_hl_lvar_match_id = matchadd(g:ruby_hl_lvar_hl_group, b:ruby_hl_lvar_match_pattern, g:ruby_hl_lvar_highlight_priority)
```

Comment:
Could you replace `0` on L62 with the option?
https://github.com/todesking/ruby_hl_lvar.vim/blob/10b6f79ecc1ba1f5cbbecd9621652cbf20e25776/autoload/ruby_hl_lvar.vim#L62

This plugin uses both of `matchadd` and `matchaddpos` functions, so we need to update both function callings.




By the way, I guess we can drop to support `matchadd` function. 
`matchaddpos` is available since Vim 7.4.330, but we can use Vim 8.1 now.
It is not a too edge API already, so we have no reason to keep compatibility of old Vim 7.4.

Reply:
Fixed b18e752b9794ca74b92a2239969ec3ccb8aa48de